actuators/actuators.go

```diff
@@ -40,9 +44,11 @@ func getRole(machine *clusterv1.Machine) string {
 }
 
 func getExternalLoadBalancerNode(clusterName string) (*nodes.Node, error) {
+	fmt.Printf("Getting external load balancer node for cluster %q\n", clusterName)
```

Comment:
These `fmt.Printf`s should be replaced with log statements now that https://github.com/kubernetes-sigs/cluster-api-provider-docker/pull/27 has merged.

Reply:
#27 added loggers into the clusterActuator and machineActuators. This function is not in either of those scopes. So, we'll need to use `klog` or something like that.
Looks like the build also failed because of `undefined: fmt`

Reply:
the interim step is to pass a log through to all of these functions and rely on the logger interface. we should not use klog anywhere.

Reply:
> we should not use klog anywhere.
+1
always pass a logger interface and not the logging implementation directly, whether it's fmt or klog.

Reply:
fixed!